todo-app/src/TodoList.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+import {observer} from 'mobx-react';
+import {todoStore} from './stores';
+
+@observer
+export default class TodoList extends React.Component {
+
+  state = {
+    text: '',
+  }; 
+
+  onAddClick = (text) => {
+    text = this.state.text
+    todoStore.addTodo(text)
+  };
+
+  onAddChange = event => {
+    var text = event.target.value;
+    this.setState({
+      text: text
+    });
+  };
+
+  onDelete = (todoID) => {
+    todoStore.onDelete(todoID); 
```

Comment:
Small thing: `todoStore.onDelete` is not an event handler, but a method on `todoStore`. Best to just call it `delete` or `deleteTodo`.